### DISCUSSION TITLE
Is there a text normalization feature in spaCy?

### DISCUSSION THREAD

**Discussion by vanderbuildt** [explosion/spaCy#4970] [New Features & Project Ideas]
## Feature description
For example: input text = "I have 3 oranges dr."
Output text: "I have three oranges doctor."

**Comment by adrianeboyd** [comment]
No, spacy doesn't have a text normalization feature. Spacy explicitly never modifies the input text. It's expected that normalization would be performed in pre- or post-processing instead.
